animeal/src/Flows/Main/Modules/Home/FeedingPointDetails/View/Extensions/FeedingPointAnnotationModel+Annotation.swift

```diff
@@ -15,7 +15,7 @@ public extension FeedingPointAnnotationModel {
     
     func annotation(at coordinates: CLLocationCoordinate2D) -> Annotation {
         var annotation = PointAnnotation(id: identifier, coordinate: coordinates)
-        annotation.image = .init(image: image, name: "\(kind)")
+        annotation.image = .init(image: image, name: "\(kind)_\(hungerLevel)")
```

Comment:
it seems that name is used by MapBox to cache images and it should be different for different hunger levels images also